content/blog/switching-the-application-stack-from-pern-to-mern/index.md

```diff
@@ -0,0 +1,392 @@
+---
+title: "Switching the application stack from PERN to MERN"
+date: 2020-09-18
+meta_desc: Demonstrating the simplicity, modularity, and reusability of running an application on Kubernetes using Pulumi.
+meta_image: meta.png
+authors: ["vova-ivanov"]
+tags: ["aws", "typescript", "docker", "kubernetes"]
+---
+
+In this blog post we will return to the PERN application we previously [migrated to Kubernetes]({{< relref "/blog/deploying-a-pern-stack-application-to-aws" >}}), and switch out the PostgreSQL database for MongoDB. Although at first glance it might seem like a difficult task, the straigthforward design of Pulumi and Kubernetes allows us to easily transition the application form a PERN stack to a MERN one.
```

Comment:
```suggestion
In this blog post, we return to the PERN application we previously [migrated to Kubernetes]({{< relref "/blog/deploying-a-pern-stack-application-to-aws" >}}) and replace the PostgreSQL database with MongoDB. Although it might seem like a difficult task initially, the straightforward design of Pulumi and Kubernetes allows us to easily transition the application form a PERN stack to a MERN one.
```